### ISSUE TITLE
No background jobs

### ISSUE THREAD

**Issue by wu07sn** [nextcloud/maps#516]
After manual scanning photos an run cron.php photos appear on the map, nut new added photos are still not on the map after several days. It seems that the os_jobs table doesn't contains a Map jobs.
How can I add the cron background job? Or is there an other problem?

**Comment by beccon4** [comment]
Could it be linked to #478?

**Comment by wu07sn** [comment]
No. It is not the same Issue. After running  occ maps:scan-photos there are photos on the map, but if new photos are added they doesn't appear on the map. If I run occ maps:scan-photos again then they are on the map.
But running occ maps:scan-photos clears all photos and add them again and it tooks a lot of time.

**Comment by pbtoubol** [comment]
Hello, I have the same problem on my NC 20... Do you have a new idea to resolve this issue ?
Thank you

**Comment by wu07sn** [comment]
Sorry I don't have any solution yet :-(

**Comment by Holger79** [comment]
Are your photos on an external storage? I remember I read about a Problem one year ago...

**Comment by Thovi98** [comment]
> 
> 
> Are your photos on an external storage? I remember I read about a Problem one year ago...

I’ve same issue, and my photos are not on an external storage...
Manual `occ maps:scan-photos` does the job, so yes it may be related to a problem with cron job.

**Comment by meierwitt** [comment]
Hi,
same problem here. It worked in December. I think it stoped after updating my server to Ubuntu 20.04 and reinstalled NC from Backup.
After I do an `occ maps:scan-photos` the pictures appears on the map. No errors in logfiles. Cron seems to work.
Thanks
  Frank

**Comment by wu07sn** [comment]
That's right, I also restored nextcloud from the backup

**Comment by jacotec** [comment]
I have the same issue. Cron job runs fine, but new photos are not added to the map. The occ command works fine, but as already mentioned it clears the database and does everything new.

The best would be a fix for the main issue, but as a workaround an occ command which only scans and processes new files would also help.

My pictures are also not on a external storage.

**Comment by GAS85** [comment]
Cron job executred, but I can't see any maps task in it. `oc_jobs` contains nothing related to maps.
